graphics/rcd/lang/en_GB.txt

```diff
@@ -185,6 +196,8 @@ strings {
 
 		// Ride build window strings.
 		RIDE_BUILD_TITLEBAR_SHOP:         "Place shop";
+		RIDE_BUILD_TITLEBAR_GENTLE:       "Place gentle ride";
+		RIDE_BUILD_TITLEBAR_THRILL:       "Place thrill ride";
```

Comment:
Apparently, we have both gentle and thrill rides. The FGTR block doesn't say if the spooky manor is gentle or thrill. Should it?

Reply:
Found it already, I missed it. Sorry.